<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Model Creation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cnn_model">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>CNN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reglin_model">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
     </font>
    </property>
    <property name="text">
     <string>Reglin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rnn_model">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>RNN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="svm_model">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
     </font>
    </property>
    <property name="text">
     <string>SVM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knn_model">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
     </font>
    </property>
    <property name="text">
     <string>KNN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kmeans_model">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
     </font>
    </property>
    <property name="text">
     <string>K-Means</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
